database/src/migrations/release.0.7/biohub.0.7.sql

```diff
@@ -0,0 +1,1871 @@
+    create_date                  timestamp(6)    DEFAULT now() NOT NULL,
+    create_user                  integer         NOT NULL,
+    update_date                  timestamp(6),
+    update_user                  integer,
+    revision_count               integer         DEFAULT 0 NOT NULL,
+    CONSTRAINT pk49_2_2_1_1 PRIMARY KEY (id)
+)
+;
+
+
+
+COMMENT ON COLUMN no_sample_permit.id IS 'System generated surrogate primary key identifier.'
+;
+COMMENT ON COLUMN no_sample_permit.number IS 'Permit number provided by FrontCounter BC.'
+;
+COMMENT ON COLUMN no_sample_permit.issue_date IS 'The date the permit was issued.'
+;
+COMMENT ON COLUMN no_sample_permit.end_date IS 'The date the permit is no longer valid.'
+;
+COMMENT ON COLUMN no_sample_permit.coordinator_first_name IS 'The first name of the permit coordinator.'
+;
+COMMENT ON COLUMN no_sample_permit.coordinator_last_name IS 'The last name of the permit coordinator.
+'
+;
+COMMENT ON COLUMN no_sample_permit.coordinator_email_address IS 'The email address.'
+;
+COMMENT ON COLUMN no_sample_permit.coordinator_agency_name IS 'The permit coordinator agency name.'
+;
+COMMENT ON COLUMN no_sample_permit.create_date IS 'The datetime the record was created.'
+;
+COMMENT ON COLUMN no_sample_permit.create_user IS 'The id of the user who created the record as identified in the system user table.'
+;
+COMMENT ON COLUMN no_sample_permit.update_date IS 'The datetime the record was updated.'
+;
+COMMENT ON COLUMN no_sample_permit.update_user IS 'The id of the user who updated the record as identified in the system user table.'
+;
+COMMENT ON COLUMN no_sample_permit.revision_count IS 'Revision count used for concurrency control.'
+;
+COMMENT ON TABLE no_sample_permit IS 'Provides a record of scientific permits for which no sampling was conducted.'
+;
+
+-- 
+-- TABLE: project 
+--
+
+CREATE TABLE project(
+    id                            integer                    GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
+    pt_id                         integer                    NOT NULL,
+    name                          varchar(50)                NOT NULL,
+    objectives                    varchar(3000)              NOT NULL,
+    management_recovery_action    character(1),
+    location_description          varchar(3000),
+    start_date                    date                       NOT NULL,
+    end_date                      date,
+    caveats                       varchar(3000),
+    comments                      varchar(3000),
+    coordinator_first_name        varchar(50)                NOT NULL,
+    coordinator_last_name         varchar(50)                NOT NULL,
+    coordinator_email_address     varchar(500)               NOT NULL,
+    coordinator_agency_name       varchar(300)               NOT NULL,
+    coordinator_public            character(1)               NOT NULL,
+    geometry                      geometry(polygon, 3005),

```

Comment:
Just double checking, is it okay to specify `polygon` here?

I don't actually know for sure what it implies, but the current version is:
```
geometry(Geometry,3005) CHECK (st_isValid(geom));
```
If I had to guess, it seems like specifying polygon might limit the kinds of geometries we can save in this column? Like maybe we wouldn't be able to save `points` (as those aren't a type of polygon?).

maybeMa question for @popkinj

Reply:
@NickPhura good question. this was what was supplied by the DA team for generating from the ER Studio tool. but it could be that we're in flux with the work that @popkinj  is doing with that team on the standards. easy enough to modify.

Reply:
Yeah... Geometry will allow all valid feature types. If you specify Polygon or Multi-Polygoon.. Points or lines would be seen as invalid.

Reply:
@NickPhura @popkinj i'm good to go ahead and adjust.